pkg/storage/stores/shipper/bloomshipper/client.go

```diff
@@ -208,29 +209,31 @@ func (c ClosableReadSeekerAdapter) Close() error {
 	return nil
 }
 
-func BlockRefFrom(tenant, table string, md v1.BlockMetadata) BlockRef {
-	return BlockRef{
-		Ref: Ref{
-			TenantID:       tenant,
-			TableName:      table,
-			Bounds:         md.Series.Bounds,
-			StartTimestamp: md.Series.FromTs,
-			EndTimestamp:   md.Series.ThroughTs,
-			Checksum:       md.Checksum,
-		},
+func newRefFrom(tenant, table string, md v1.BlockMetadata) Ref {
```

Comment:
nit: newRefFrom is only called once, and it's to immediately pass the ref into newBlockRefWithEncoding; should the functions be combined?

Reply:
I did the separation of concerns and composition of the functions intentionally. No strong opinions, though.

Reply:
Yeah I don't feel strongly about it either, Let's keep what you did IMO